Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/OverridePropertyHandler.cpp

```diff
@@ -10,24 +10,56 @@
 
 namespace AzToolsFramework::Prefab
 {
-    OverridePropertyHandler::OverridePropertyHandler()
+    PrefabOverrideLabelHandler::PrefabOverrideLabelHandler()
+        : m_overridden(false)
+        , m_iconButton(new QToolButton())
+        , m_textLabel(new AzQtComponents::ElidingLabel())
     {
+        QHBoxLayout* layout = new QHBoxLayout(this);
+        layout->setMargin(0);
+        layout->setSpacing(0);
+
+        layout->addWidget(m_iconButton);
+        layout->addWidget(m_textLabel);
+
+        m_textLabel->setProperty("overridden", false);
```

Comment:
constexpr this string since this be used in multiple places and possibly tests would need access too.